overlord/snapstate/snapmgr.go

```diff
@@ -305,11 +305,14 @@ type SnapState struct {
 	// in the snap are removed from this list on link-snap, so that we can
 	// remember services that were disabled in another revision and then renamed
 	// or otherwise removed from the snap in a future refresh.
```

Comment:
does this comment need updates?

Reply:
I added a small separate comment for the user one, but the comment should be good